src/components/modules/document/common/DocumentType/modules/DocumentStatus/modules/StatusVerified.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div>
+    <span
+      v-if="date"
+      class="document-status-verified-expire-date"
+    >
+      {{ date | formatUnixDate('MMM DD, YYYY') }}
```

Comment:
Move format to filter, or create special filter for this purpose

Reply:
for us enought `formatUnixDate`, no need to create special filter with special format